drivers/windows/R/web.R

```diff
@@ -36,10 +36,10 @@ web_client <- R6::R6Class(
     },
 
     submit = function(path, name, resources, cluster = NULL,
-                      depends_on = NULL) {
+                      depends_on = NULL, workdir = "") {
```

Comment:
What is going on with workdir here? I'm not thrilled that we need to set it on linux but not windows and that is has a default -- which presumably won't work on linux